packages/dialtone-css/lib/build/less/components/scrollbar.less

```diff
@@ -0,0 +1,25 @@
+.custom-scrollbars {
```

Comment:
If `.custom-scrollbars` is **our** name for it, maybe it should be named `.dt-custom-scrollbars` — unless it's a selector from the plugin.

Reply:
This one I forgot to change to our new name, thanks 👍